scripts/linux/runTests.sh

```diff
@@ -20,6 +20,10 @@ BUILD_CONFIG="$2"
 SUFFIX='Microsoft.Azure*test.dll'
 OUTPUT_FOLDER=$BUILD_BINARIESDIRECTORY
 
+echo 'dotnet version:' 
+dotnet --version
+/opt/hostedtoolcache/dotnet/dotnet --version
```

Comment:
reminder: remove dotnet version visibility or clean this up to be more descriptive